ccloud-observability/docs/observability-overview.rst

```diff
@@ -51,11 +51,11 @@ The following instructions will:
 
 -  use ``ccloud-stack`` to create a |ccloud| cluster, a service account with proper acls, and a client configuration file
 
--  create a ``cloud`` resource api-key for the ``ccloud-exporter``
+-  create a ``cloud`` resource api-key for accessing the Metrics API /export endpoint
 
 -  build a |ak| client docker image with the maven project's dependencies cache
 
--  stand up numerous docker containers (1 consumer with JMX exporter, 1 producer with JMX exporter, Prometheus, Grafana, a ccloud-exporter, and a node-exporter) with ``docker-compose``
+-  stand up numerous docker containers (1 consumer with JMX exporter, 1 producer with JMX exporter, Prometheus, Grafana and a node-exporter) with ``docker-compose``
```

Comment:
@javabrett What is a "node-exporter"?

Reply:
> @javabrett What is a "node-exporter"?

It's a reference to an instance of this that we are deploying:

- https://prometheus.io/docs/guides/node-exporter/
- https://github.com/prometheus/node_exporter

Good for collecting machine/host metrics and exposing them as a Prometheus scrapeable endpoint.  Nothing Confluent or Kafka related here.

Reply:
Do users need to know this?  If yes, would be good to hyperlink to sites you mentioned above.?

Reply:
Done.